jbmc/src/java_bytecode/ci_lazy_methods.cpp

```diff
@@ -255,6 +223,52 @@ bool ci_lazy_methodst::operator()(
   return false;
 }
 
+/// Look for virtual callsites with no candidate targets. If we have
+/// invokevirtual A.f and we don't believe either A or any of its children
+/// may exist, we assume specifically A is somehow instantiated. Note this
+/// may result in an abstract class being classified as instantiated, which
+/// stands in for some unknown concrete subclass: in this case the called
+/// method will be a stub.
+/// \return whether a new class was encountered
+bool ci_lazy_methodst::handle_virtual_methods_with_no_callees(
+  std::unordered_set<irep_idt> &methods_to_convert_later,
+  std::unordered_set<irep_idt> &instantiated_classes,
+  const std::unordered_set<exprt, irep_hash> &virtual_function_calls,
+  symbol_tablet &symbol_table)
+{
+  bool any_new_classes;
```

Comment:
Give default value! Currently if `virtual_function_calls` is empty it is UB what this returns